roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml

```diff
@@ -126,19 +126,12 @@
     - xpack-security
     - molecule-idempotence-notest
 
-- name: Delete certs.zip in Generator node
```

Comment:
This task should be kept

Reply:
if this .zip file is deleted then it will be regenerated on every run and a new certificate will be created, isn't it?